crypto/message.go

```diff
@@ -228,14 +228,25 @@ func (msg *PGPMessage) getEncryptionKeyIDs() ([]uint64, bool) {
 	packets := packet.NewReader(bytes.NewReader(msg.Data))
 	var err error
 	var ids []uint64
+	var encryptedKey *packet.EncryptedKey
 	for {
 		var p packet.Packet
 		if p, err = packets.Next(); err == io.EOF {
 			break
 		}
-		enc, ok := p.(*packet.EncryptedKey)
-		if ok {
-			ids = append(ids, enc.KeyId)
+		switch p := p.(type) {
+		case *packet.EncryptedKey:
+			encryptedKey = p
+			ids = append(ids, encryptedKey.KeyId)
+		case *packet.SymmetricallyEncrypted:
+			break
+		case *packet.AEADEncrypted:
+			break
+		case *packet.Compressed:
+			break
+		case *packet.LiteralData:
+			break
```

Comment:
```suggestion
		case *packet.SymmetricallyEncrypted:
		case *packet.AEADEncrypted:
		case *packet.Compressed:
		case *packet.LiteralData:
			break
```

Reply:
Golang does not have default fallthrough:  In a case or default clause, the last non-empty statement may be a (possibly labeled) "fallthrough" statement to indicate that control should flow from the end of this clause to the first statement of the next clause. Otherwise control flows to the end of the "switch" statement. A "fallthrough" statement may appear as the last statement of all but the last clause of an expression switch.

Reply:
Ah yeah, sorry. But maybe you can do this:
```
		case *packet.SymmetricallyEncrypted, *packet.AEADEncrypted, *packet.Compressed, *packet.LiteralData:
			break
```

Reply:
yeah, through maybe using simpler case condition is making it easier to understand at first glance.

Reply:
Yeah, but I think repeating the case body 4 times is also a bit ugly, especially if we're gonna have this code in multiple places. If you want, you could split up the four case values above across 4 lines, to make it easier to read, though :man_shrugging: